### ISSUE TITLE
Add fastline file structure for F-Droid

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#206]
Your gains by using Fastlane:

- you can maintain summary/description in your app's repo and will not need to open MRs each time you change them
- you can provide screenshots which will be shown along with the description, raising the attention of potential users
- you can even provide per-release changelogs, which will be shown inside the app's description, so your users don't have to "hunt them down"

TL;DR: while not mandatory, it's highly recommended for your F-Droid listings.

https://gitlab.com/snippets/1895688
